src/Incidents/components/IncidentTypes/index.js

```diff
@@ -67,10 +83,66 @@ class IncidentTypes extends Component {
     this.setState({ showFilters: false });
```

Comment:
Why setting showFilters false here while the initialState of this component has show filters false?